tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
That's interesting.  FlxTest#resetGame() calls FlxG.resetGame(), which eventually should mean hitting FlxG.reset(), which calls input.reset(), which ought to, ya know, reset the inputs, I guess.  But actually FlxMouse's reset() just resets the mouse button states.  I guess that makes sense, otherwise you might have a weird cursor jump for the user, but I'm not sure how resetGame() gets used in the wild anyway.